src/parser/jobs/rpr/modules/OtherGauges.tsx

```diff
@@ -0,0 +1,142 @@
+import {Trans} from '@lingui/react'
+import Color from 'color'
+import {DataLink} from 'components/ui/DbLink'
+import {Action, ActionKey} from 'data/ACTIONS'
+import {JOBS} from 'data/JOBS'
+import {Event, Events} from 'event'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Actors} from 'parser/core/modules/Actors'
+import {CounterGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+type GaugeModifier = Partial<Record<Event['type'], number>>
+
+const SUGGESTION_TIERS = {
+	10: SEVERITY.MINOR,
+	30: SEVERITY.MEDIUM,
+	50: SEVERITY.MAJOR,
+}
+
+const SHROUD_ACTIONS: ActionKey[] = [
+	'GIBBET',
+	'GALLOWS',
+	'GUILLOTINE',
+	'PLENTIFUL_HARVEST',
+	'ENSHROUD',
+]
+
+const GAUGE_FADE = 0.25
+
+const DESIGN_GAIN = 10
```

Comment:
nit: this name confused the shit out of me before I went down and looked for where you used it.